content/compose/multiple-compose-files/_index.md

```diff
@@ -1,20 +1,20 @@
 ---
 description: General overview for the different ways you can work with multiple compose
   files in Docker Compose
-keywords: compose, compose file, merge, extends, include, docker compose
+keywords: compose, compose file, merge, extends, include, docker compose, -f flag
 title: Overview
 ---
 
 {{< include "compose-eol.md" >}}
 
-This section contains information on the three key ways you can use multiple Compose files in your Compose application. 
+This section contains information on the ways you can work with multiple Compose files. 
 
-Using multiple Compose files lets you customize a Compose application for different environments or workflows. This is useful for large applications that may use dozens of containers, with ownership distributed across multiple teams. 
+Using multiple Compose files lets you customize a Compose application for different environments or workflows. This is useful for large applications that may use dozens of containers, with ownership distributed across multiple teams. For example, if your organization or team uses a monorepo, each team may have their own “local” Compose file to run a subset of the application. They then need to rely on other teams to provide a reference Compose file that defines the expected way to run their own subset. Complexity moves from the code in to the infrastructure and the configuration file.
 
-For example, if your organization or team uses a monorepo, each team may have their own “local” Compose file to run a subset of the application. They then need to rely on other teams to provide a reference Compose file that defines the expected way to run their own subset. Complexity moves from the code in to the infrastructure and the configuration file.
+The quickest way to work with multiple Compose files is to [merge](merge.md) Compose files, using the `-f` flag in the command line to list out your desired Compose files. However, [merging rules](merge.md#merging-rules) means this can soon get quite complicated.
```

Comment:
nit
```suggestion
The quickest way to work with multiple Compose files is to [merge](merge.md) Compose files using the `-f` flag in the command line to list out your desired Compose files. However, [merging rules](merge.md#merging-rules) means this can soon get quite complicated.
```